### PR TITLE
Logs and events fixes

### PR DESCRIPTION
**Pull request by mshitrit**
[ECOPROJECT-1826](https://issues.redhat.com//browse/ECOPROJECT-1826), [ECOPROJECT-1827](https://issues.redhat.com//browse/ECOPROJECT-1827), [ECOPROJECT-1829](https://issues.redhat.com//browse/ECOPROJECT-1829)
Fixing some minor bugs improving SNR logs and events

### CODE REVIEW

**Comment by razo7** [comment]
/lgtm
It surprises me that adding the finalizer is not one of the first things SNR does, but when it adds later then your code changes make sense :)

/hold
Giving others a chance to get more reviews (feel free to un-hold)

File: controllers/selfnoderemediation_controller.go [hunk 1]
```diff
@@ -483,6 +485,7 @@ func (r *SelfNodeRemediationReconciler) recoverNode(node *v1.Node, snr *v1alpha1
 			return ctrl.Result{}, err
 		}
 		r.Recorder.Event(snr, eventTypeNormal, eventReasonRemoveFinalizer, "Remediation process - remove finalizer from snr")
```
Comment (clobrano) [inline controllers/selfnoderemediation_controller.go:487]: Shouldn't this use `events.NormalEvent` from `common`?

Comment (razo7) [reply controllers/selfnoderemediation_controller.go:487]: IIRC the common events PR have been added after SNR events PR was merged. I guess we didn't update SNR with the common events API

Comment (clobrano) [reply controllers/selfnoderemediation_controller.go:487]: I was asking becase in the line below we use `events.RemediationFinished`, which should come from the common events API

Comment (mshitrit) [reply controllers/selfnoderemediation_controller.go:487]: Good point, TBH I didn't think of that .
I've only considered the "special" events.
I'll update in another PR

Comment (mshitrit) [reply controllers/selfnoderemediation_controller.go:487]: Here: https://github.com/medik8s/self-node-remediation/pull/178

**Pull request merged**